content/blog/blog-2020-07-building-your-first-operator-2.md

```diff
@@ -0,0 +1,633 @@
+
+Now we can deploy again, and check the service has been correctly created. 
+
+```
+operator $ kubectl kudo plan status --instance galera-instance
+Plan(s) for "galera-instance" in namespace "default":
+.
+└── galera-instance (Operator-Version: "galera-0.1.0" Active-Plan: "deploy")
+    └── Plan deploy (serial strategy) [COMPLETE], last updated 2020-06-24 14:50:51
+        └── Phase deploy (serial strategy) [COMPLETE]
+            ├── Step bootstrap_config [COMPLETE]
+            ├── Step bootstrap_service [COMPLETE]
+            ├── Step bootstrap_deploy [COMPLETE]
+            ├── Step firstboot_config [COMPLETE]
+            └── Step cluster_services [COMPLETE]
+
+operator $ kubectl get services
+NAME                            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                               AGE
+galera-instance-bootstrap-svc   ClusterIP   None         <none>        3306/TCP,4444/TCP,4567/TCP,4568/TCP   2m51s
+galera-instance-hs              ClusterIP   None         <none>        3306/TCP,4444/TCP,4567/TCP,4568/TCP   2m28s
+MacBook-Pro:operator matt$ kubectl describe service galera-instance-hs
+Name:              galera-instance-hs
+Namespace:         default
+Labels:            app=galera
+                   galera=galera-instance
+                   heritage=kudo
+                   kudo.dev/instance=galera-instance
+                   kudo.dev/operator=galera
+Annotations:       kudo.dev/last-applied-configuration:
+                     {"kind":"Service","apiVersion":"v1","metadata":{"name":"galera-instance-hs","namespace":"default","creationTimestamp":null,"labels":{"app"...
+                   kudo.dev/last-plan-execution-uid: 0374f5bc-27e9-4b28-ada9-371a149cf05b
+                   kudo.dev/operator-version: 0.1.0
+                   kudo.dev/phase: deploy
+                   kudo.dev/plan: deploy
+                   kudo.dev/step: cluster_services
+Selector:          app=galera,instance=galera-instance
+Type:              ClusterIP
+IP:                None
+Port:              mysql  3306/TCP
+TargetPort:        3306/TCP
+Endpoints:         <none>
+Port:              sst  4444/TCP
+TargetPort:        4444/TCP
+Endpoints:         <none>
+Port:              replication  4567/TCP
+TargetPort:        4567/TCP
+Endpoints:         <none>
+Port:              ist  4568/TCP
+TargetPort:        4568/TCP
+Endpoints:         <none>
+Session Affinity:  None
+Events:            <none>
+```
+
+We can see here that the additional service has been created correctly, is using the correct selectors, and it currently has no endpoints since we haven’t deployed any nodes yet. 
+
+Once again, let’s clean up the cluster and move on to our next step. 
+
+Now we have our configuration and service defined, we can finally add our StatefulSet. Once again, add the steps, tasks and resources to our `operator.yaml`: 
+
+```yaml
+# operator.yaml
+steps:
+          - name: statefulset
+            tasks:
+              - statefulset
```

Comment:
```suggestion
        - name: statefulset
          tasks:
            - statefulset
```

This seems to have a wrong indentation - Not sure if my suggestion really fixes it.